### ISSUE TITLE
Ng add command not working on Angualr 11

### ISSUE THREAD

**Issue by devhus** [valor-software/ng2-bootstrap#6067]
Bug description:
- When trying to install the package using ``ng add ngx-bootstrap `` it shows an error.
- ``An unhandled exception occurred: Cannot find module '@schematics/angular/utility/config'``

ngx-bootstrap:
- default
Angular:
- 11
Bootstrap:
- both
Build system: Angular CLI, System.js, webpack, starter seed:
Angalur CLI
**Expected behavior**
```
[error] Error: Cannot find module '@schematics/angular/utility/config'
Require stack:
- C:\xampp\htdocs\bnaturalia\bnaturalia-frontend\node_modules\ngx-bootstrap\schematics\ng-add\index.js
- C:\xampp\htdocs\bnaturalia\bnaturalia-frontend\node_modules\@angular-devkit\schematics\tools\export-ref.js
- C:\xampp\htdocs\bnaturalia\bnaturalia-frontend\node_modules\@angular-devkit\schematics\tools\index.js
- C:\xampp\htdocs\bnaturalia\bnaturalia-frontend\node_modules\@angular\cli\utilities\json-schema.js
- C:\xampp\htdocs\bnaturalia\bnaturalia-frontend\node_modules\@angular\cli\models\command-runner.js
- C:\xampp\htdocs\bnaturalia\bnaturalia-frontend\node_modules\@angular\cli\lib\cli\index.js
- C:\Users\devhus\AppData\Roaming\npm\node_modules\@angular\cli\lib\init.js
- C:\Users\devhus\AppData\Roaming\npm\node_modules\@angular\cli\bin\ng
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (C:\xampp\htdocs\bnaturalia\bnaturalia-frontend\node_modules\ngx-bootstrap\schematics\ng-add\index.js:11:18)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at new ExportStringRef (C:\xampp\htdocs\bnaturalia\bnaturalia-frontend\node_modules\@angular-devkit\schematics\tools\export-ref.js:19:25)
    at SchematicEngineHost._resolveReferenceString (C:\xampp\htdocs\bnaturalia\bnaturalia-frontend\node_modules\@angular-devkit\schematics\tools\node-module-engine-host.js:95:21)
    at SchematicEngineHost._resolveReferenceString (C:\xampp\htdocs\bnaturalia\bnaturalia-frontend\node_modules\@angular\cli\models\schematic-engine-host.js:62:22)
    at SchematicEngineHost.createSchematicDescription (C:\xampp\htdocs\bnaturalia\bnaturalia-frontend\node_modules\@angular-devkit\schematics\tools\file-system-engine-host-base.js:174:34)
    at SchematicEngine.createSchematic (C:\xampp\htdocs\bnaturalia\bnaturalia-frontend\node_modules\@angular-devkit\schematics\src\engine\engine.js:215:38)
```

**Comment by beletate** [comment]
Try `npm uninstall -g @angular/cli` then `npm install -g @angular/cli`.

**Comment by Jan-Ole** [comment]
@beletate that does not work for me. I'm on Version 11.2.9 for cli and schematics which is currently the latest version and I get the same error.

**Comment by rendaardy** [comment]
Related issue #6037

**Comment by valorkin** [comment]
closed as duplicate of #6037 
watch for https://github.com/valor-software/ngx-bootstrap/pull/6167
and release of ngx-bootstrap v7 in the nearest days
